### DISCUSSION TITLE
s/niqztauj16pbq3i05lsuca

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#2] [Announcements]
# s/niqztauj16pbq3i05lsuca

最好用的流程编辑器bpmn-js系列之基本使用

https://blog.ops-coffee.cn/s/niqztauj16pbq3i05lsuca

**Comment by fengyinchao** [comment]
非常好的教程，感谢！

**Comment by QiMengChao** [comment]
很详细

**Comment by 944189562** [comment]
bpmn-js 入坑系列
